utils/build/docker/dotnet/weblog/app.csproj

```diff
@@ -48,7 +48,6 @@
         <PackageReference Include="RabbitMQ.Client" Version="6.4.0"/>
         <PackageReference Include="MongoDB.Driver" Version="2.23.1"/>
 
-        <PackageReference Include="Datadog.Trace" Version="*" Condition="'$(Configuration)' != 'Release'" />
-        <Reference Include="/opt/datadog/netcoreapp3.1/Datadog.Trace.dll" Condition="'$(Configuration)' == 'Release'" />
+        <PackageReference Include="Datadog.Trace" Version="*" />
```

Comment:
Don't we want to use `/opt/datadog/netcoreapp3.1/Datadog.Trace.dll` instead to test unpublished builds (`dev` vs `prod`)?